diff --unified --recursive --text termite/termite.cc termite-moddedd/termite.cc
--- termite/termite.cc	2019-11-14 16:10:55.976986978 -0500
+++ termite-patched/termite.cc	2019-11-14 16:13:44.783669902 -0500
@@ -1638,7 +1638,6 @@
 
 int main(int argc, char **argv) {
     GError *error = nullptr;
-    const char *const term = "xterm-termite";
     char *directory = nullptr;
     gboolean version = FALSE, hold = FALSE;
 
@@ -1825,8 +1824,6 @@
     }
 #endif
 
-    env = g_environ_setenv(env, "TERM", term, TRUE);
-
     GPid child_pid;
     if (vte_terminal_spawn_sync(vte, VTE_PTY_DEFAULT, nullptr, command_argv, env,
                                 G_SPAWN_SEARCH_PATH, nullptr, nullptr, &child_pid, nullptr,
